index.js

```diff
@@ -4,18 +4,22 @@ const ESC = '\u001B[';
 const isTerminalApp = process.env.TERM_PROGRAM === 'Apple_Terminal';
 
 x.cursorTo = function (x, y) {
-	if (arguments.length === 0) {
-		return ESC + 'H';
+	if (x !== 'number') {
```

Comment:
`typeof`?

Reply:
Yes, I was distracted when pushing that...

Reply:
Oeps, didn't notice Vadim already commented that :).